egs/wsj/s5/steps/lmrescore.sh

```diff
@@ -98,6 +98,8 @@ case "$mode" in
      # grammar and transition weights.
     mdl=`dirname $indir`/final.mdl
     [ ! -f $mdl ] && echo No such model $mdl && exit 1;
+    [[ -f `dirname $indir`/frame_subsampling_factor && $self_loop_scale != 1.0 ]] && \
+      echo "WARNING: chain models need '--self-loop-scale 1.0'";
```

Comment:
please in echo statements, start them with 'echo "$0:' so we know which script produces the message.